Whiskey/Functions/Install-WhiskeyPowerShellModule.ps1

```diff
@@ -93,57 +93,74 @@ function Install-WhiskeyPowerShellModule
     {
         $modulesRoot = $Path
     }
-    $moduleRoot = Join-Path -Path $modulesRoot -ChildPath $Name
-    $moduleManifestPath = Join-Path -Path $moduleRoot -ChildPath ('{0}\{1}.psd1' -f $moduleVersionDirName,$Name)
 
-    $manifest = $null
-    $manifestOk = $false
-    try
+    if(Test-WhiskeyPowershellModule -Name $Name -Version $Version)

```

Comment:
Move this check to [Install-WhiskeyTool](https://github.com/webmd-health-services/Whiskey/blob/0b4ef44d0b5cb55a7909d34fb1298aac687dcaa9/Whiskey/Functions/Install-WhiskeyTool.ps1#L128-L140). This function should always install a module to the local build root when it's called. We want to use `Test-WhiskeyPowerShellModule` to determine _if_ we call `Install-WhiskeyPowerShellModule`, not _how_ it works.

Also it means we won't affect the `GetPowerShellModule` task as it calls this function to install modules. That task should always install modules into the local build root, regardless if they're installed globally (at least with its current design). This is actually the reason for the build failure with these changes.